pkg/reconciler/service/resources/configuration_test.go

```diff
@@ -127,3 +129,14 @@ func TestInlineConfigurationSpec(t *testing.T) {
 		t.Errorf("expected %q labels got %q", want, got)
 	}
 }
+
+func TestConfigurationHasNoKubectlAnnotation(t *testing.T) {
+	s := createServiceWithKubectlAnnotation()
+	c, err := makeConfiguration(s)
+	if err != nil {
+		t.Errorf("unexpected error: %v", err)
+	}
+	if v, ok := c.Annotations[corev1.LastAppliedConfigAnnotation]; ok {
+		t.Errorf("expected missing %q annotation got %q", corev1.LastAppliedConfigAnnotation, v)
```

Comment:
We mostly do: "Got before want".
```suggestion
		t.Errorf("Annotation %s = %q, want: empty", corev1.LastAppliedConfigAnnotation, v)
```